src/Ssh/Client.php

```diff
@@ -81,6 +85,9 @@ public function run(Host $host, string $command, array $config = [])
         }
 
         $ssh = "ssh $sshArguments $host $become 'bash -s; printf \"[exit_code:%s]\" $?;'";
+        if ($this->output->isDebug()) {
```

Comment:
not sure whether this is required in this case, as the process below uses this `pop->callback()`...